master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -190,7 +192,13 @@ private[celeborn] class Master(
   }
   masterSource.addGauge(MasterSource.IS_ACTIVE_MASTER) { () => isMasterActive }
 
+  threadPoolSource.registerSource("master-noneager-handler", nonEagerHandler)
+  threadPoolSource.registerSource(
+    "netty-rpc-connection",
```

Comment:
Could the pattern of thread pool name be unified?

Reply:
Done